### PR TITLE
Remove public nav from login/register page and submission form (first step in removing public all together)

### PR DESCRIPTION
**Pull request by theskumar**
- Remove primary nav and convert auth pages to apply template
- Move SystemSettings as generic singleton setting
- remove link to fund detail page from listing
- ~Remove mailchimp~ reverted
- ~Remove Google Translate widget~ reverted

### CODE REVIEW

**Comment by theskumar** [comment]
@frjo removed the links from the listing page.

**Comment by frjo** [comment]
Deploying this to test now.

**Review by frjo — APPROVED** [review]

**Comment by frjo** [comment]
@theskumar We have some conflicts with main, can you please rebase this PR and the one that builds on top of it?

**Comment by theskumar** [comment]
> @theskumar We have some conflicts with main, can you please rebase this PR and the one that builds on top of it?

Done, also fixed the code that was causing the migration to fail: https://github.com/HyphaApp/hypha/pull/3432/commits/a7341c36668095230d758f857d3db14170a65616

**Review by frjo — APPROVED** [review]

**Comment by theskumar** [comment]
updated application form to use the base apply instead of public -> base. 

The single sand fund

File: hypha/apply/funds/templates/funds/application_base.html [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
-        <ul>
-            {% for error in form.non_field_errors %}
-                <li class="error">{{ error }}</li>
-            {% endfor %}
-        </ul>
-    {% else %}
-    <h5 class="heading heading--no-margin heading--regular">{% trans "There were some errors with your form. Please amend the fields highlighted below" %}</h5>
+<h1 class="mt-8">{{ page.title }}</h1>
+
+<div class="wrapper wrapper--form">
```
Comment (frjo) [inline hypha/apply/funds/templates/funds/application_base.html]: I think the "wrapper--medium" class is better to keep here.

**Review by frjo — CHANGES_REQUESTED** [review]

File: hypha/apply/funds/templates/funds/application_base.html (hunk 1 above)
Comment (theskumar) [reply hypha/apply/funds/templates/funds/application_base.html]: Let me give that a try and see how that looks.

Comment (theskumar) [reply hypha/apply/funds/templates/funds/application_base.html]: Updated. https://github.com/HyphaApp/hypha/pull/3432/commits/76db0a180df7977a927aa52ebef5ab3b40736641

Screenshot 2023-07-20 at 06 32 27

**Comment by theskumar** [comment]
Updated so that we keep mailchimp, google translate and also fixed the issue where the public nav was not visible for some of the public pages.

**Review by frjo — APPROVED** [review]

**Pull request merged**